kernel/vmbytegen.ml

```diff
@@ -900,7 +900,8 @@ let compile_constant_body ~fail_on_error env univs = function
         | _ ->
             let evar_expand _ = assert false in
             let evar_relevance _ = assert false in
-            let sigma = { evar_expand = evar_expand; evar_relevance } in
+            let evar_repack _ = assert false in
+            let sigma = { evar_expand; evar_relevance; evar_repack } in
```

Comment:
This could be defined in constr.ml as eg `Constr.evar_expand_noevars`

Reply:
This is actually `Constr.default_evar_handler`. I'll use that one.